package/boot/uboot-envtools/files/ipq40xx

```diff
@@ -32,7 +32,8 @@ ubootenv_mtdinfo () {
 
 case "$board" in
 openmesh,a42 |\
-openmesh,a62)
+openmesh,a62 |\
+glinet,gl-b1300)
```

Comment:
please stick to the lexicographical ordering and move the glient,gl-b1300 in front of the openmesh,a42. Thanks.

Reply:
Fixed, thanks.